tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -477,7 +477,7 @@ function getPackageType(tree: Tree, options: NormalizedSchema) {
 function isJs(tree: Tree, options: NormalizedSchema) {
   const jsSourceFiles: string[] = [];
   visitNotIgnoredFiles(tree, options.paths.sourceRoot, treePath => {
-    if (treePath.includes('.js') || treePath.includes('.jsx')) {
```

Comment:
was the problem that this check detected `package.json` as having `.js` ?

Reply:
yes sir ( package.json, tsconfig.json )